src/webhook/github.js

```diff
@@ -30,6 +30,40 @@ class WebhookHandler {
         return crypto.timingSafeEqual(Buffer.from(signature), Buffer.from(digest));
     }
 
+    static async recalculateRanks(){
+        const users = await prisma.users.findMany({
+            orderBy: {
+              points: 'desc',
+            },
+            select: {
+              githubId: true,
+              points: true,
+            },
+          });
+        
+          let currentRank = 0;
+          let lastPoints = null;
+          let usersToUpdate = [];
+        
+          users.forEach((user, index) => {
+            if (lastPoints !== user.points) {
+              currentRank += 1;
+              lastPoints = user.points;
+            }
+            usersToUpdate.push({
```

Comment:
you are pushing all users whether new rank is equals or not to old rank, you can put a condition to check if both are unequal then only push.